cli/translate.py

```diff
@@ -0,0 +1,126 @@
+import sys
+from collections import OrderedDict
+from typing import Any, Dict, List
+
+from datastore.reader.core import GetAllRequest, GetRequest, Reader
+from datastore.reader.services import register_services as register_reader_services
+from datastore.shared.di import injector
+from datastore.shared.util import fqid_from_collection_and_id
+from datastore.writer.core import RequestUpdateEvent, Writer, WriteRequest
+from datastore.writer.services import register_services as register_writer_services
+
+from openslides_backend.locale.translator import Translator
+from openslides_backend.models.models import Organization
+from openslides_backend.shared.util import ONE_ORGANIZATION_FQID
+
+collection_to_fields_map = OrderedDict(
+    {
+        "organization": [
+            "name",
+            "login_text",
+            "description",
+        ],
+        "meeting": [
+            "name",
+            "welcome_title",
+            "welcome_text",
+            "motion_preamble",
+            "motions_export_title",
+            "assignments_export_title",
+            "users_pdf_welcometitle",
+            "users_pdf_welcometext",
+            "users_email_sender",
+            "users_email_subject",
+            "users_email_body",
+        ],
+        "group": ["name"],
+        "motion_workflow": ["name"],
+        "motion_state": ["name", "recommendation_label"],
+        "projector_countdown": ["name"],
+        "projector": ["name"],
+        "motion": ["recommendation_label"],
+    }
+)
+possible_languages = Organization().default_language.constraints["enum"]
+
+
+def read_collection(collection: str, fields: List[str]) -> Any:
+    reader: Reader = injector.get(Reader)
+    with reader.get_database_context():
+        response = reader.get_all(
```

Comment:
use `DeletedModelsBehaviour.NO_DELETED` to not fetch deleted models.

Reply:
Okay, I have added this option to the get_all, but I thought it is the default.